Gems/Atom/Feature/Common/Assets/Shaders/LightCulling/LightCulling.azsl

```diff
@@ -253,7 +269,7 @@ void CullDecals(uint groupIndex, TileLightData tileLightData, float3 aabb_center
         // ATOM-4224 - try AABB-AABB 
         float boundingSphereRadiusSqr = dot(decal.m_halfSize, decal.m_halfSize);
           
-        bool potentiallyIntersects = TestSphereVsAabb(decalPosition, boundingSphereRadiusSqr, aabb_center, aabb_extents);
+        bool potentiallyIntersects = TestSphereVsAabb(decalPosition, boundingSphereRadiusSqr, aabb_center, aabb_extents / 2.0);
```

Comment:
nit: there is a semantic mismatch between BuildAabb defined in NVLC and here. In the former `aabb_extensts` are halfExtents, in the latter, those are full extents (consistent with the AZ::Aabb). You might consider renaming to aabb_halfExtents to avoid confusion and remove repeating division by 2.0.
On the other hand, I am not sure if the BuildAabb in NVLC is still needed after #18516

Reply:
Thanks for pointing it out.
I decided not to change variable naming to reduce code change. Now in the whole file, var name is consistent `aabbExtent` are logically `aabb_halfExtents`.